### ISSUE TITLE
[Proposal] : allow additional customization of "in memoriam" with an Icon/emoji/badge

### ISSUE THREAD

**Issue by playmobilmeister** [mastodon/mastodon#25618]
### Pitch

# td;dr version
**Not critical** but hopefully easy to implement and will contribute to communities: Customize the way that deceased members are memorialized to allow more visibility and community-feeling, by allowing an instance to customize some memorialization method(s). At minimum allow the customization of the "In Memoriam" message (including standard Emoji like 🕯)
# Background
As far as I can tell from 
[this issue](https://github.com/mastodon/mastodon/issues/23570) and [this PR](https://github.com/mastodon/mastodon/pull/23614), when a profile status becomes "In Memoriam", the way it is marked in the web UI is a sentence that reads "In Memoriam". This is already helpful and I hope that that merged PR will affect us soon. 
# Problem
- It seems to me that this is not visible enough in both profile page and in conversations.
- The expression "In memoriam" Does not translate well into all cultures and languages. Many would probably prefer "RIP" , "Legacy account", "in memory of {username}", to name just a few examples.

I would like to propose some additional stuff, which I hope would not be too difficult to implement. I don't mean that all of these options would be available, as I don't want to trouble the programmers too much about this. just 2 or 3 of these would already be huge improvements as a start.

roles affected: Admin or Admin + Moderator

# My Proposals:

## [Server Wide - controlled by admins]:

1. Customizing the message content (e.g. instead of "In Memoriam", allow for "Rest in Peace, Friend").
   - If Emoji are allowed that would be great. even standard emoji like 🕯 or 🪦 would help greatly.
   - If a server's custom emoji can be used, that would already solve much of the problem, I believe.
   - If fields like the Username can be refereed to, that would be awesome ! e.g. ("May you flower forever, {username}. We shall remember you" for a community of plant lovers)

2. Customizing the font size and color for that existing message, so that people don't miss it. I am guessing this one is the most easy to implement. is it ?

3. Adding a colored tag or even just additional text that appears before or after the user name. For example, "Playmobilmeister" becomes "Playmobilmeister [RIP]", where the "[RIP]" part is in a different color, font, or bold/italic

4. Allowing the frame that envelopes a user's icon to be specified as dotted instead of full, or with a different thickness or color.
4. Allowing a special tag/badge to be created out of Emoji/Unicode/ico/png. This can be displayed as
    - a big icon on top of the profile banner (say, an icon of a tombstone or a candle) and/or
    - a little badge to the side of every instance of the profile icon, so that this appears in conversations. 

5. If 3 is too difficult then like 3 but non - customizable i.e. just choices from a list of emoji or unicode options or some icons.

Any of the above would be greatly appreciated !

## [Per Profile - controlled by admins and moderators]
Allow The customization of the above w.r.t. to the chosen server default, so as to allow a more personalized message that will express the appreciation of the particular contribution of the deceased member.

### Motivation

- More visibility for the fact that a profile belonged to a deceased community member and is not active. The text is not noticeable enough by some
- personalized appreciation of the deceased in the message
- Expressing the values of the Community within the message

**Comment by nehemiagurl** [comment]
I agree that there's room for improvement even beyond the "in memoriam" banner. I think that even something simple like this mockup could be very effective:
![218994709-9aa27fa9-17a0-40aa-9b27-815920443854](https://github.com/mastodon/mastodon/assets/109424320/0b1352fd-e98e-4946-95d8-c527b4aecf4e)
